recipes/etcd_cluster.rb

```diff
@@ -145,7 +150,12 @@
   template "#{node['cookbook-openshift3']['etcd_conf_dir']}/etcd.conf" do
     source 'etcd.conf.erb'
     notifies :restart, 'service[etcd]', :immediately
-    variables etcd_servers: etcd_servers
+    variables lazy {
```

Comment:
this is a new technique for me - do these only get assigned if lazy is referenced?